tests/CMakeLists.txt

```diff
@@ -1,4 +1,25 @@
-find_package(GTest REQUIRED)
+find_package(GTest)
+if (NOT GTest_FOUND)
+  message(STATUS "No installed GTest found, fetching from Github")
+  include(FetchContent)
+  FetchContent_Declare(
+    googletest
+    GIT_REPOSITORY https://github.com/google/googletest.git
+    GIT_TAG        7153098229e88295f9655ff1d3b0e2fa9eada5f8
```

Comment:
Is there a way to attach this to a particular release tag, rather than a SHA1?  Thanks! : - )

Reply:
I'll rephrase: Could you please change this to fetch from the current latest release?  Thanks! : - )